frontend/test/pytest/test_mcmc.py

```diff
@@ -37,6 +37,6 @@ def test_mcmc():
         )
     )
     def circuit():
-        return qml.state()
+        return qml.expval(qml.X(0))
 
     print(circuit.mlir)
```

Comment:
This test doesn't even do anything. I don't think coverage is a good enough reason to add a test that checks nothing (contrary to what the comment says), would you mind removing it? :) 
If this results in coverage issues ignore.